app/core/BackgroundBridge.js

```diff
@@ -118,13 +119,18 @@ export class BackgroundBridge extends EventEmitter {
 		const publicState = this.getProviderNetworkState(memState);
```

Comment:
nit: we could destructive here
```suggestion
		const { chainId, networkVersion } = this.getProviderNetworkState(memState);
```
and then just refer to those instead